Any project - nil pointer dereference (DwarfReader)

On any project (including simple hello world projects) I get the following exception:
```dlv debug
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xc0000005 code=0x0 addr=0x30 pc=0x626c2c]

goroutine 1 [running]:
panic(0x8d6380, 0xc04200c0b0)
        C:/Go/src/runtime/panic.go:500 +0x1af
github.com/derekparker/delve/pkg/proc.(*EvalScope).DwarfReader(0xc0429dd440, 0xc04202a800)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/pkg/proc/variables.go:324 +0x2c
github.com/derekparker/delve/pkg/proc.(*EvalScope).packageVarAddr(0xc0429dd440, 0x967909, 0x14, 0x1808440ea1, 0xc0423d8050, 0xbd27f8)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/pkg/proc/variables.go:663 +0x3c
github.com/derekparker/delve/pkg/proc.(*Process).EvalPackageVariable(0xc042191680, 0x967909, 0x14, 0x1, 0x0, 0x40, 0x0, 0x0, 0xc042207500, 0xc04202f620, ...)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/pkg/proc/variables.go:654 +0x8b
github.com/derekparker/delve/pkg/proc.(*Process).getGoInformation(0xc042191680, 0x0, 0x0, 0x0, 0x0, 0x0, 0xc04201e000, 0xc042140320, 0xc042197eb8)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/pkg/proc/proc.go:894 +0xc3
github.com/derekparker/delve/pkg/proc.initializeDebugProcess(0xc042191680, 0xc042199130, 0x4c, 0x0, 0xc04202a800, 0xc04202ab68, 0xc000000000)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/pkg/proc/proc.go:834 +0xd7
github.com/derekparker/delve/pkg/proc.newDebugProcess(0xc042191680, 0xc042199130, 0x4c, 0x0, 0x0, 0xc0421996d0)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/pkg/proc/proc_windows.go:156 +0x352
github.com/derekparker/delve/pkg/proc.Launch(0xc04213e0f0, 0x1, 0x1, 0x959f75, 0x1, 0x0, 0x0, 0x0)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/pkg/proc/proc_windows.go:118 +0x6e7
github.com/derekparker/delve/service/debugger.New(0xc04219e510, 0xc04219e510, 0xc0420739f0, 0x40f73f)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/service/debugger/debugger.go:69 +0x2be
github.com/derekparker/delve/service/rpccommon.(*ServerImpl).Run(0xc042199090, 0xc042199000, 0xc042199090)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/service/rpccommon/server.go:117 +0x144
github.com/derekparker/delve/cmd/dlv/cmds.execute(0x0, 0xc04213e0f0, 0x1, 0x1, 0xc042138600, 0x1, 0x0)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/cmd/dlv/cmds/commands.go:416 +0x1ed
github.com/derekparker/delve/cmd/dlv/cmds.debugCmd.func1(0xc042156a00, 0xbe4eb0, 0x0, 0x0, 0x0)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/cmd/dlv/cmds/commands.go:243 +0x4ca
github.com/derekparker/delve/cmd/dlv/cmds.debugCmd(0xc042156a00, 0xbe4eb0, 0x0, 0x0)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/cmd/dlv/cmds/commands.go:244 +0x50
github.com/derekparker/delve/vendor/github.com/spf13/cobra.(*Command).execute(0xc042156a00, 0xbe4eb0, 0x0, 0x0, 0xc042156a00, 0xbe4eb0)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/vendor/github.com/spf13/cobra/command.go:569 +0x418
github.com/derekparker/delve/vendor/github.com/spf13/cobra.(*Command).ExecuteC(0xc042156400, 0xc042156a00, 0xc042156800, 0xc042156600)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/vendor/github.com/spf13/cobra/command.go:656 +0x36e
github.com/derekparker/delve/vendor/github.com/spf13/cobra.(*Command).Execute(0xc042156400, 0xc042073f70, 0xc04201e000)
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/vendor/github.com/spf13/cobra/command.go:615 +0x32
main.main()
        C:/Users/kdeenanauth/Documents/git/go/src/github.com/derekparker/delve/cmd/dlv/main.go:13 +0x5c```



1. What version of Delve are you using (`dlv version`)? `0.12.1`
2. What version of Go are you using? (`go version`)? `1.7.5`
3. What operating system and processor architecture are you using? `windows/amd64`
4. What did you do? `dlv debug`

--- a/pkg/proc/proc_windows.go
+++ b/pkg/proc/proc_windows.go
@@ -241,7 +241,7 @@ func (dbp *Process) addThread(hThread syscall.Handle, threadID int, attach, susp
 	}
 	thread.os.hThread = hThread
 	dbp.threads[threadID] = thread
-	if dbp.CurrentThread == nil {
+	if dbp.currentThread == nil {
 		dbp.SwitchThread(thread.ID)
 	}
 	if suspendNewThreads {

